cmd/root.go

```diff
@@ -253,9 +257,32 @@ See each command's help for details on how to use the generated script.`, rootCm
 }
 
 // setFlagsFromConfig reads in the config file if it exists and uses any flag values for commands.
-func setFlagsFromConfig() {
-	viper.SetConfigFile(config.GetConfigFile())
+func setFlagsFromConfig() error {
+	configFile := config.GetConfigFile()
+	viper.SetConfigType("yml")
+	viper.SetConfigFile(configFile)
+
+	err := makePath(configFile)
```

Comment:
This creates the `.config/ldcli` directory if it doesn't exist so viper can write the file.